x-pack/plugin/sql/qa/src/main/resources/agg.sql-spec

```diff
@@ -563,5 +566,7 @@ SELECT MIN(salary) min, MAX(salary) max, gender g, languages l, COUNT(*) c FROM
 // group by with literal
 implicitGroupByWithLiteral
 SELECT 10, MAX("salary") FROM test_emp;
+literalWithGroupBy
+SELECT 1 FROM test_emp GROUP BY gender;
```

Comment:
Here too, could you please add one more integ test with more than 1 literal and also scalars involved?